### PR TITLE
Added parameter LRO in order to mark LRO

### PR DESCRIPTION
**Pull request by Alancere**

### CODE REVIEW

File: eng/tools/internal/exports/exports.go [hunk 1]
```diff
@@ -54,6 +54,8 @@ type Func struct {
 
 	// a comma-delimited list of the return types
 	Returns *string `json:"returns,omitempty"`
+
```
Comment (tadelesh) [inline eng/tools/internal/exports/exports.go:57]: Please add a comment about the param.

Comment (Alancere) [reply eng/tools/internal/exports/exports.go:57]: comment: ”mark long running operation“ is it okay？

Comment (tadelesh) [reply eng/tools/internal/exports/exports.go:57]: could you please provide some examples of this string?

Comment (Alancere) [reply eng/tools/internal/exports/exports.go:57]: image

**Review by tadelesh — APPROVED** [review]

**Pull request merged**